apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/SerialConsistencyValidator.java

```diff
@@ -0,0 +1,21 @@
+package io.stargate.sgv2.api.common.config;
+
+import io.stargate.bridge.proto.QueryOuterClass;
+import java.util.Arrays;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class SerialConsistencyValidator
+    implements ConstraintValidator<SerialConsistencyTypeAnnotation, QueryOuterClass.Consistency> {
+  private QueryOuterClass.Consistency[] subset;
+
+  @Override
+  public void initialize(SerialConsistencyTypeAnnotation constraint) {
+    this.subset = constraint.anyOf();
```

Comment:
* can `anyOf` be null in any case?
* transform to list already here (or sort the array and use binary serach in the `isValid` if you wanna be fancy ;)